docs/materials/sw/part2-ex3-python.md

```diff
@@ -132,15 +132,15 @@ We now have our Python installation and our Python script - we just need to writ
 		:::bash
 		#!/bin/bash
 
-		tar xzf prebuilt_python.tar.gz 
+		tar -xzf prebuilt_python.tar.gz 
```

Comment:
FYI, both work. Personally, I tend to use `tar` without the hyphen. But this is fine.

Reply:
Just trying to be consistent with previous instructions.  It's minor though.  I know both work.